include/zephyr/pm/pm.h

```diff
@@ -72,6 +72,31 @@ struct pm_notifier {
 	const void *ctx;
 };
 
+/**
+ * @brief Helper macro for building a pm_notifiers list name.
+ *
+ * @param node_id Power states child node identifer
+ */
+#define Z_PM_NOTIFIERS_NAME(node_id) UTIL_CAT(pm_notifiers_, DT_NODE_CHILD_IDX(node_id))
+
+/**
+ * @brief Generate a pm_notifiers list for the passed power-states child node
+ *	  identifier.
+ *
+ * @param node_id CPU node identifier
+ */
+#define PM_NOTIFIERS_CREATE_STATES(node_id)				\
```

Comment:
all DT specific APIs should contain `_DT_` to make it explicit that it has to be used with DT

also, why is this one exposed publicly?

Reply:
Added `_DT_` and `Z_` prefix.